src/Codeception/Subscriber/Console.php

```diff
@@ -527,7 +527,9 @@ protected function detectWidth()
         ) {
             $this->width = (int) (`command -v tput >> /dev/null 2>&1 && tput cols`) - 2;
         } else if ($this->isWin() && (php_sapi_name() == "cli")) {
-            $this->width = 200;
+            exec('mode con', $output);
+            preg_match('/^ +.* +(\d+)$/', $output[4], $matches);
+            $this->width = (int) $matches[1] - 2;
```

Comment:
I was not sure if adding this -2 was required
I saw that for linux / mac calculation added this -2
If you tell me the meaning of this -2 maybe i could check if it is required to arbitrary reduce the detected width on Windows or not